eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -125,6 +125,12 @@
         "new": "(interface|enum|class) reactor\\..*",
         "justification": "We allow reactor classes in our external APIs. This also occurs because we check transitive dependencies, so it checks reactor jars."
       },
+      {
+        "regex": true,
+        "code": "java\\.class\\.externalClassExposedInAPI",
+        "new": "(interface|enum|class) reactor\\..*",
+        "justification": "We allow reactor classes in our external APIs. This also occurs because we check transitive dependencies, so it checks reactor jars."
+      },
```

Comment:
Is this reactor exclusion required? Looks like only gson version was updated.

Reply:
It is actually a dupe of an existing one. So, yes it is required but I do not need add another one of those here. I missed removing it when I chose to put this one in the end :)